docs/CHANGELOG.asciidoc

```diff
@@ -47,6 +47,61 @@ Review important information about the {kib} 8.x releases.
 
 Review the following information about the {kib} 8.8.0 release.
 
+[float]
+[[known-issues-8.8.0]]
+=== Known issues
+
+// tag::known-issue-155203[]
+[discrete]
+.Monitors in Synthetics may stop running
+[%collapsible]
+====
+*Details* +
+If Monitor Management was enabled prior to 8.6.0, the API key generated internally will not contain the required permissions. The Synthetics app will attempt to fix this automatically in {kibana-pull}155203[#155203] when a user with https://www.elastic.co/guide/en/observability/8.8/synthetics-role-setup.html[sufficient privileges] visits this page for the first time after upgrading to 8.8.0.
+
+*Impact* +
+All monitors configured to run on Elastic's global managed testing infrastructure will stop running until a user with permissions has loaded the Synthetics app.
+====
+// end::known-issue-155203[]
+
+// tag::known-issue-156798[]
+[discrete]
+.Network throttling disabled for browser monitors in Synthetics
+[%collapsible]
+====
+*Details* +
+Network throttling has been temporarily disabled for browser-based Synthetics monitors running on Elastic's global managed testing infrastructure and private locations. This will be enabled again at some point in the future. We're providing frequent updates on this issue in https://github.com/elastic/synthetics/blob/main/docs/throttling.md[this document].
+
+*Impact* +
+With network throttling being disabled, your monitors may run more quickly (i.e. have a lower duration) than you observed previously and than when network throttling is enabled again in the future. No monitor configurations have been changed, but the network throttling settings are ignored at the moment.
+====
+// end::known-issue-156798[]
+
+// tag::known-issue-120[]
+[discrete]
+.Alert failures when migrating to 8.8.0 from 8.6 or earlier
+[%collapsible]
+====
+*Details* +
+If a cluster meets all of the following conditions, its {elastic-sec} and {observability} rules will fail and no actions will be sent:
+
+- The {elastic-sec} and {observability} rules were created in version 8.6 or earlier releases.
+- There must be an {ref}/index-templates.html[index template] (for any index) that isn't composed of component templates.
+
+The following error messages in the {kib} log occur when {kib} starts or when the rules run:
+
+[source,sh]
+----
+Error installing component template .alerts-ecs-mappings - Cannot read properties of undefined (reading 'includes')
+
+Error installing common resources for AlertsService. No additional resources will be installed and rule execution may be impacted. - Failure during installation. Cannot read properties of undefined (reading 'includes')
+----
+
+*Impact* +
```

Comment:
I would consider calling this a 'fix' or 'solution', as opposed to 'impact'... Don't block on this, though.

Reply:
That's a template that's used in all the known issues, so I'll stick with it for now.